### ISSUE TITLE
Thumbnails/Minimap for PDFs

### ISSUE THREAD

**Issue by sameersingh** [izuzak/atom-pdf-view#185]
It'll be great for a scrollable navigation column (a sidebar) on the right showing the thumbnails of each page, allowing us to quickly scroll through large PDFs and get to the part we're looking for.

**Comment by izuzak** [comment]
Yeah, that would be neat, @sameersingh -- thanks for opening an issue! ⭐️ I probably won't have time to implement this myself, but happy to review a pull request if someone else wants to do it! 👍

**Comment by izuzak** [comment]
> If you open a PDF in Firefox (which also uses PDF.js), it automatically displays a minimap. would it not be possible just to "unhide" the extra UI?

@sbrl I don't know. I'd be okay if you wanted to try to do that yourself and opened a pull request 👍

**Comment by sbrl** [comment]
I'm very busy at the moment (University semester time!), but I _may_ have time to investigate in a few months time.
